frontend/angular-js/README.md

```diff
@@ -1,17 +1,26 @@
 # AngularJS Tech Guides [Tech Guides under construction]
 
-## Kickoff Guide
+## Content
 
-At [Wolox](http://wolox.com.ar), we have developed the [Angular kickoff guide](./docs/kickoff/README.md) to create our Angular projects. Check it out to find a detailed guide to start with your project.
+- [Project Bootstrapper](#project-bootstrapper)
+- [Style Guide](#style-guide)
+- [Open Source](#open-source)
+- [Useful Documentation](#useful-documentation)
+
+## Project Bootstrapper
+
+At [Wolox](http://wolox.com.ar), we have developed the [Angular Bootstrap](http://github.com/wolox/frontend-bootstrap) (use `angular` branch) to create our Angular projects. You'll find a detailed guide to start with your project in its README.
+
+## Style Guide
+
+We use the following [Style Guide](https://github.com/johnpapa/angular-styleguide) as reference for our Angular style projects.
 
 ## Open Source
 
-- [Wolox Front-End Bootstrap](http://github.com/wolox/frontend-bootstrap) (use `angular` branch)
 - [Custom Multiple Select](https://codepen.io/SKOLZ/pen/PJqPGG)
```

Comment:
move this to the front-end readme.

Reply:
You mean to add an `open source` section in the front end main page ?